.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+    "name": "Python 3.11 Bookworm",
+    // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+    // "image": "mcr.microsoft.com/devcontainers/typescript-node:0-20",
+    "build": {
+        "dockerfile": "Dockerfile"
```

Comment:
Curious, why don't we use the root level dockerfile and instead use the newly added one? If we remove the CMD "bash" in the root level dockerfile, will this work?

Reply:
The root dockerfile copies your local code into the container as part of the build. This is different to devcontainers which mounts the code into the container. Best to keep it seperate till I can confirm everything works great in devcontainers also.